### PR TITLE
[All Hosts] (debugging) Preference for the debugging tool vs. network share deployment

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Adds new article that mostly (but not entirely) replaces the old article about deploying on a Network Share. Updates links appropriately. 

THIS IS AN OLD PR AND I HAVEN'T UPDATED MOST OF THE ms.date VALUES.

### CODE REVIEW

File: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ You can test an Office Add-in in an Office client that's on Windows by publishin
 > Deployment by network share isn't supported for production add-ins. This method has the following limitations.
 >
 > - The add-in can only be installed on Windows computers.
+> - Add-ins that use the [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md) are not supported by publishing to a network share.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md:16]:
```suggestion
> - Add-ins that use the [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md) aren't supported when published to a network share.
```

File: docs/testing/test-debug-non-local-server.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Test and debug Office Add-ins on a non-local server
+description: Learn how to test and debug your Office Add-in on a non-local host.
+ms.date: 07/12/2024
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/test-debug-non-local-server.md:4]: Before you merge, remember to update the ms.date in the various files

```suggestion
ms.date: 08/08/2024
```

File: docs/testing/test-debug-non-local-server.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+> [!IMPORTANT]
+> The office-addin-debugging tool registers the add-in in the Windows registry or a special folder on a Mac. For an Outlook add-in, it also registers the add-in in Exchange. To avoid subtle bugs when developing, always end a testing session by running `npm run stop` to ensure that these registrations are removed and that the server process is fully stopped. *Manually closing the server, the command line window (or TERMINAL), Visual Studio Code, or the Office application doesn't remove these registrations.*
+
+## Other projects
+
+If your project wasn't created with one of Teams Toolkit or Yo Office, run the tool with npx in the root of the project. Invoke it with its `start` command followed by the relative path to the manifest. The following is an example:
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/test-debug-non-local-server.md:24]:
```suggestion
If your project wasn't created with Teams Toolkit or Yo Office, run the tool with npx in the root of the project. Invoke it with its `start` command followed by the relative path to the manifest. The following is an example.
```

File: docs/testing/test-debug-non-local-server.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+```command&nbsp;line
+npx office-addin-debugging start manifest.json
+```
+
+This command sideloads the add-in for testing and debugging. The tool also works with an XML manifest.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/test-debug-non-local-server.md:30]: should we use the "add-in only manifest" term here?

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

File: docs/testing/test-debug-non-local-server.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Other projects
+
+If your project wasn't created with Teams Toolkit or Yo Office, run the tool with npx in the root of the project. Invoke it with its `start` command followed by the relative path to the manifest. The following is an example.
+
+```command&nbsp;line
+npx office-addin-debugging start manifest.json
+```
+
+This command sideloads the add-in for testing and debugging. The tool also works with an XML manifest.
```
Comment (Rick-Kirkham) [inline docs/testing/test-debug-non-local-server.md:30]:
```suggestion
This command sideloads the add-in for testing and debugging. The tool also works with an add-in only manifest.
```

**Pull request merged**